ParisBNF/abb/BNFabb99.xml

```diff
@@ -0,0 +1,324 @@
+                  </acquisition>
+               </history>
+                  <additional>
+                     <adminInfo>
+                        <recordHist>
+                           <source>
+                              <listBibl type="catalogue">
+                                   <bibl>
+                                      <ptr target="bm:ContiRossini1914Notice"/>
+                                      <citedRange unit="page">61</citedRange>
+                                      <citedRange unit="number">33</citedRange>
+                                      </bibl>
+                                   <bibl>
+                                      <ptr target="bm:Chaine1912Catalogue"/>
+                                      <citedRange unit="page">63</citedRange>
+                                      <citedRange unit="number">99</citedRange>
+                                   </bibl>
+                                   <bibl>
+                                      <ptr target="bm:Abbadie1859Cat"/>
+                                      <citedRange unit="page">110</citedRange>
+                                      <citedRange unit="number">99</citedRange>
+                                   </bibl>
+                                    </listBibl>
+
+                           </source>
+                        </recordHist>
+                        </adminInfo>
+                     <listBibl type="secondary">
+                        <bibl>
+                        <ptr target="bm:KnibbUllendorff1978Enoch1"/>
+                     </bibl>
+                     <bibl>
+                     <ptr target="bm:KnibbUllendorff1978Enoch2"/>
+                  </bibl>
+                     </listBibl>
+                  </additional>
+
+                  </msDesc>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <projectDesc>
+            <p>Encoded according to TEI P5 Guidelines.</p>
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <particDesc>
+           <listPerson>
+               <person>
+                 <persName ref="PRS1071dAbbadi" role="owner"/>
+              </person>
+              <person>
+                 <persName ref="PRS1071dAbbadi" role="bequeather"/>
+                 </person>
+
+            </listPerson>
+         </particDesc>
+         <textClass>
+            <keywords>
+               <term key="Bible"/>
+               <term key="Apocrypha"/>
+               <term key="ChristianLiterature"/>
+            </keywords>
+         </textClass>
+         <langUsage><language ident="en">English</language>
+         <language ident="gez">Gǝʿǝz</language></langUsage>
```

Comment:
amharic is also used in the record and must be defined here

Reply:
ok